### PR TITLE
changelog: add missing LP bug refs for 27.3

### PR DESCRIPTION
**Pull request by orndorffgrant**
## Proposed Commit Message

```
changelog: add missing LP bug refs for 27.3
```
```
changelog: update ua-timer.timer frequency
```

## Test Steps

Ensure these are the correct bugs and associated with the correct lines in the changelog

## Desired commit type::

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist:

 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -5,10 +5,10 @@ ubuntu-advantage-tools (27.3~21.10.1) impish; urgency=medium
     - purge old ua-messaging.timer/service files
   * systemd:
     - remove ua-messaging.timer/service
-    - add new ua-timer.timer that runs every 2 hours
+    - add new ua-timer.timer that runs every 6 hours
```
Comment (renanrodrigo) [inline debian/changelog:8]: Thanks, my bad

File: debian/changelog [hunk 2]
```diff
@@ -22,7 +22,7 @@ ubuntu-advantage-tools (27.3~21.10.1) impish; urgency=medium
       + print warning about generic fips kernel if cloud-id fails
     - cloud:
       + rely only on cloud-id to determine cloud type (LP: #1940131)
-      + catch errors when determining cloud type (LP: #1938207) (GH: #1541)
+      + catch errors when determining cloud type (LP: #1938207, LP: #1944676) (GH: #1541)
```
Comment (renanrodrigo) [inline debian/changelog:25]: damn this was my bad as well, I included this in the SRU bug and forgot about changelog :|

**Review by renanrodrigo — APPROVED** [review]

File: debian/changelog (hunk 1 above)
Comment (orndorffgrant) [reply debian/changelog:8]: All is good, I missed it too :)

File: debian/changelog (hunk 2 above)
Comment (orndorffgrant) [reply debian/changelog:25]: All is good, I saw the addition to the bug and forgot to add it until now :)

**Comment by orndorffgrant** [comment]
Jenkins failures are flakes during reboot steps. We should look into how to make reboots during integration tests more reliable at some point. I added a jira ticket here https://warthogs.atlassian.net/browse/SC-474

**Pull request merged**